app/uk/gov/hmrc/gform/sharedmodel/formtemplate/ComponentType.scala

```diff
@@ -403,6 +403,31 @@ object MiniSummaryList {
   implicit val format: Format[MiniSummaryList] = derived.oformat()
 }
 
+case class TableValue(value: SmartString, cssClass: Option[String], colspan: Option[Int])
+
+object TableValue {
+  implicit val leafExprs: LeafExpr[TableValue] = (path: TemplatePath, r: TableValue) =>
+    r match { case t: TableValue => LeafExpr(path + "value", t.value) }
```

Comment:
This seems to be identical

```suggestion
    LeafExpr(path + "value", r.value)
```

Reply:
done